tests/MongoDB.Driver.Tests/Specifications/change-streams/ChangeStreamTestRunner.cs

```diff
@@ -402,12 +417,28 @@ private void AssertChangeStreamDocumentPropertyValuesAgainstBackingDocument(Chan
                 clusterTime.Should().BeNull();
             }
 
+            var operationType = actualDocument.OperationType;
+            operationType.ToString().ToLowerInvariant().Should().Be(backingDocument["operationType"].AsString);
+
+            if (operationType == ChangeStreamOperationType.Invalidate)
+            {
+                return;
+            }
+
             var collectionNamespace = actualDocument.CollectionNamespace;
             collectionNamespace.DatabaseNamespace.DatabaseName.Should().Be(backingDocument["ns"]["db"].AsString);
             collectionNamespace.CollectionName.Should().Be(backingDocument["ns"]["coll"].AsString);
 
             var documentKey = actualDocument.DocumentKey;
-            documentKey.Should().Be(backingDocument["documentKey"].AsBsonDocument);
+            var backingDocumentOperationType = backingDocument["operationType"]; // should be replaced on "operationType" as soon as "Rename" and "Drop" options will be implemented
```

Comment:
the current implementation doesn' have `Drop` and `Rename` change stream operation types. I've added them for local tests, but I don't think that I can leave them (I've excluded them from the commit). 

I've found only one c# ticket which is about adding `Drop` type. This is `CSHARP-2362`. As for `Rename` type, I found only the server ticket `SERVER-35028`.

So, currently, the tests for `Drop` and `Rename` types have been skipped via `skipReason`. I guess these tests should be turned on after implementation of `2362` and a c# ticket for `Rename` change stream type. 

Also, it would be better to replace `backingDocument["operationType"]` on `operationType` in the selected line above as soon as the expected tickets will be implemented. I will write appropriated comments into the expected tickets.

Not sure, maybe there is a better solution to solve this situation. Thoughts?